### ISSUE TITLE
stash list in the sidebar 

### ISSUE THREAD

**Issue by StevenKuo** [rowanj/gitx#424]
Version 0.15.1964 dev (0.15.1964)

under OSX 10.10,  I can't see stash list in the sidebar.
I can see the branch ref/stash, but it just show stash@{0}, I want to see the others stash detail.

**Comment by toxsid** [comment]
Agreed. I would like this feature as well, maybe it should not be considered a feature since it was available in GitX(L) but a regression in this fork.

**Comment by obrejla** [comment]
+1 It would be really great, stash is musthave ;) Thanks!
